hedy.py

```diff
@@ -788,7 +788,8 @@ class ConvertToPython_8(ConvertToPython_7):
     def for_loop(self, args):
         args = [a for a in args if a != ""]  # filter out in|dedent tokens
         all_lines = [indent(x) for x in args[3:]]
-        return "for " + args[0] + " in range(" + "int(" + args[1] + ")" + ", " + "int(" + args[2] + ")+1" + "):\n"+"\n".join(all_lines)
+        step = "1 if int(" + args[1] + ")" + " < " + "int(" + args[2] + ") else -1"
```

Comment:
I know we also don't always use them but an f-string would be a bit more readable here (see: https://realpython.com/python-f-strings/)